trunk/lib/utils/wc-yotpo-settings-functions.php

```diff
@@ -45,12 +53,22 @@ function wc_proccess_yotpo_settings() {
           'reviews_widget' => $widgets_instances['reviews_widget'],
           'qna' => $widgets_instances['qna'],
           'star_rating' => $widgets_instances['star_rating'],
+          'reviews_carousel' => $widgets_instances['reviews_carousel'],
+          'promoted_products' => $widgets_instances['promoted_products'],
+          'reviews_tab' => $widgets_instances['reviews_tab'],
       ),
       'v3_widgets_enables' => array(
           'reviews_widget_product' => isset($_POST['yotpo_reviews_widget_enabled_product']) ? true : false,
           'qna_product' => isset($_POST['yotpo_qna_widget_enabled_product']) ? true : false,
           'star_rating_product' => isset($_POST['yotpo_star_rating_enabled_product']) ? true : false,
           'star_rating_category' => isset($_POST['yotpo_star_rating_enabled_category']) ? true : false,
+          'reviews_carousel_product' => isset($_POST['yotpo_reviews_carousel_enabled_product']) ? true : false,
+          'reviews_carousel_category' => isset($_POST['yotpo_reviews_carousel_enabled_category']) ? true : false,
+          // 'reviews_carousel_home' => isset($_POST['yotpo_reviews_carousel_enabled_home']) ? true : false,
```

Comment:
remove?

Reply:
right, it won't be needed in the nearest future